src/calendar.h

```diff
@@ -203,6 +203,10 @@ class time_duration
             return time_duration( t );
         }
         template<typename T>
+        static constexpr time_duration from_moves( const T t ) {
+            return time_duration( t / 100 );
+        }
```

Comment:
We've got a couple of places that could use this, search `from_moves` in the current repo.

Reply:
i will look into it.
should i do it in the same pr or wait for this one to be merged and make a seperate one?